packages/next/test/metrics.test.js

```diff
@@ -15,7 +15,7 @@ test('should configure metrics correctly with both node and http metrics', async
   const { url } = await createRuntime(t, configuration)
 
   // This is needed for the diagnostics channel to start intercepting requests
-  await sleep(100)
+  await sleep(LOGS_TIMEOUT)
```

Comment:
from 100 to a few seconds seems a big increase

Reply:
It's probably an overestimation. I'll lower to 5000 (in the CI) and 1000 (locally).
Anyway, we had differently values over the codebase and I tried to unify them.

Reply:
Trying